seed/static/seed/js/services/analyses_service.js

```diff
@@ -0,0 +1,27 @@
+angular.module('BE.seed.service.analyses', [])
+  .factory('analyses_service', [
+    '$http',
+    '$log',
+    'user_service',
+    function (
+      $http,
+      $log,
+      user_service
+    ) {
+
+      let get_labels = function() {
```

Comment:
Thinking this might be a typo, but I'd avoid the term `labels` since SEED has the widely-used [Status]Label model.

Actually, maybe we don't even need this method? For the analyses GET list endpoint, we should always use the explicitly provided org_id as opposed to a user's "currently chosen" organization as indicated at the top right of the page.

Reply:
I honestly think I just forgot to remove this function from a reference copy-paste.  It's not being used atm!